sdks/python/apache_beam/runners/worker/sdk_worker.py

```diff
@@ -464,16 +496,17 @@ class SdkWorker(object):
 
   def __init__(self,
                bundle_processor_cache,  # type: BundleProcessorCache
-               state_cache_metrics_fn=list,
+               state_cache_metrics_fn=list,  # type: Callable[[], Iterable[metrics_pb2.MonitoringInfo]]
                profiler_factory=None,  # type: Optional[Callable[..., Profile]]
-               log_lull_timeout_ns=None,
+               log_lull_timeout_ns=None,  # type: Optional[int]
               ):
+    # type: (...) -> None
     self.bundle_processor_cache = bundle_processor_cache
     self.state_cache_metrics_fn = state_cache_metrics_fn
     self.profiler_factory = profiler_factory
     self.log_lull_timeout_ns = (
         log_lull_timeout_ns or DEFAULT_LOG_LULL_TIMEOUT_NS)
-    self._last_full_thread_dump_secs = 0
+    self._last_full_thread_dump_secs = 0.0
```

Comment:
this attribute eventually gets set to the value of `time.time()` which is a `float`.